integration-test/background/grades.js

```diff
@@ -0,0 +1,82 @@
+/* global dbg:false */
+const harness = require('../helpers/harness')
+
+const tests = [
+    { url: 'duckduckgo.com', siteGrade: 'A', enhancedGrade: 'A' },
+    { url: 'theguardian.com', siteGrade: 'D-', enhancedGrade: 'B+' },
+    { url: 'google.com', siteGrade: 'D', enhancedGrade: 'D' },
+    { url: 'amazon.com', siteGrade: 'D-', enhancedGrade: 'C+' },
+    { url: 'facebook.com', siteGrade: ['D', 'C+'], enhancedGrade: 'C+' },
```

Comment:
Was seeing failures with this one - it switched between having a single blocked tracker and none at all. I figured I might as well allow a bit of a leeway when determining the site grade.